### PR TITLE
Move expectJobWithSuspendedAndNodeSelectors to test and remove util.go

### PR DESCRIPTION
**Pull request by mszadkow**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Cleanup VS code reporting issue and move `expectJobWithSuspendedAndNodeSelectors` to the test file.
#### Which issue(s) this PR fixes:

Relates to #3480

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/tas/tas_test.go [hunk 1]
```diff
@@ -45,6 +46,8 @@ const (
 	extraResource         = "example.com/gpu"
 )
 
+// +kubebuilder:docs-gen:collapse=Imports
+
```
Comment (mbobrovskyi) [inline test/e2e/tas/tas_test.go:50]: I think we don't need it.

**Comment by mimowo** [comment]
/approve
LGTM, leaving final tagging to @mbobrovskyi in case of conflicts with the other PR

**Comment by mbobrovskyi** [comment]
@mszadkow Please resolve conflicts.

**Pull request merged**